sector-builder-ffi/examples/simple/plumbing.rs

```diff
@@ -13,7 +13,7 @@ pub(crate) unsafe fn generate_post(
     ptr: *mut sector_builder_ffi_SectorBuilder,
     challenge_seed: [u8; 32],
     proving_set: &ProvingSet,
-) -> Vec<u8> {
+) -> std::result::Result<Vec<u8>, sector_builder_ffi_FCPResponseStatus> {
```

Comment:
why the `std::result::Result` prefix?

Reply:
given that these all have the same Error type, would be nicer to define a type alias, like `FfiResult`

Reply:
fixed